src/main/java/com/github/cypher/gui/root/settings/SettingsPresenter.java

```diff
@@ -21,9 +22,82 @@
 	private Executor executor;
 
 	@FXML
+	private ChoiceBox<String> languageChoiceBox;
+
+	@FXML
+	private CheckBox systemTrayCheckBox;
+
+	@FXML
+	private RadioButton enterRadioButton;
+
+	@FXML
+	private RadioButton ctrlEnterRadioButton;
+
+	@FXML
 	private Button logoutButton;
 
 	@FXML
+	private Label changesRequireRestartLabel;
+
+	@FXML
+	private void initialize() {
+		switch (settings.getLanguage().getLanguage()) {
+			case "en":
+				languageChoiceBox.setValue("English");
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/Locale.html#getDisplayLanguage(java.util.Locale) might be a better option for higher compatibility